frontend/src/static/js/v_authorship.js

```diff
@@ -210,7 +211,7 @@ window.vAuthorship = {
     },
 
     sortFiles() {
-      this.sortingFunction = (a, b) => (this.toReverseSortFiles ? -1 : 1)
+      this.sortingFunction = (a, b) => (this.sortByOrder === 'descending' ? -1 : 1)
```

Comment:
all these seems like unnecessary change when u can just bind the value true or false to `toReverseSortFiles`;
changing a binary value option to string compare is opening up spaces for bugs like typos/errors

Reply:
The `value` property of an HTML `option` element can only be a string. 

So if we want to do something like this - I will have to change the dropdown back to a checkbox. Let me know how you want me to proceed.

Reply:
Do look up `v-bind:`, take note of the syntax that we can utilize.

Reply:
Just checked. I have accordingly made the changes. Thanks for guiding